src/twisted/internet/testing.py

```diff
@@ -500,6 +544,12 @@ def __init__(self):
 
         self.readers = set()
         self.writers = set()
+        self.nameResolver = ImmediateResolver()
+
+    def installNameResolver(self, resolver: IHostnameResolver) -> IHostnameResolver:
```

Comment:
Maybe add a hint to the interface to which this method belongs to.

This is a hint that this is a production API and not some API used to support testing

```suggestion
    def installNameResolver(self, resolver: IHostnameResolver) -> IHostnameResolver:
        """
        See L{IReactorPluggableNameResolver}
        """
```

Reply:
> This is a hint that this is a production API and not some API used to support testing

OK, I'll add some context here.